src/amo/css/inc/vars.scss

```diff
@@ -14,7 +13,9 @@ $accent-color: #ebebeb;
 $breakpoints: (
   small: '(max-width: 499px)',
   medium: '(min-width: 500px)',
-  large: '(min-width: 720px)'
+  large: '(min-width: 720px)',
+  desktop: only screen and (min-width: 720px),
+  mobile: only screen and (max-width: 719px),
```

Comment:
fwiw I've switched to declaring only medium and large.

Having a mobile breakpoint is an anti-pattern; if we want to do mobile-first then we shouldn't have a way to target only mobile 😄